corehq/apps/cloudcare/api.py

```diff
@@ -183,8 +184,13 @@ def keys():
                 for bool in status_to_closed_flags(self.status):
                     yield [self.domain, owner_id, bool]
 
-        view_results = CommCareCase.view('hqcase/by_owner', keys=keys,
-                                         include_docs=False, reduce=False)
+        view_results = CommCareCase.view(
+            'hqcase/by_owner',
+            keys=keys,
+            include_docs=False,
+            reduce=False,
+            stale=settings.COUCH_STALE_QUERY
+        )
```

Comment:
let's move this function to a `dbaccessors` module either way.

i think if we can smartly workaround the issue that @dannyroberts pointed out for your own session we can change this (fine if you don't get updates made by others).

No idea off the top of my head whether that's totally doable or basically impossible. I think it would involve having some other bit of state (maybe a variable in the user session with recently touched cases?)